packages/react-dom/src/test-utils/FizzTestUtils.js

```diff
@@ -120,4 +140,59 @@ function mergeOptions(options: Object, defaultOptions: Object): Object {
   };
 }
```

Comment:
This function doesn't really need to exist. The function name is longer than the syntax to write this out. This is strictly worse since it made me look up this definition, thinking it was something more involved, but it also drops the types because it's effectively "any" typed.